console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -959,11 +978,21 @@ const fetchViewDefinition = (viewName, isRedirect) => {
   };
 };
 
-const deleteViewSql = viewName => {
+const deleteViewSql = (viewName, viewType) => {
   return (dispatch, getState) => {
     const currentSchema = getState().tables.currentSchema;
+    const property = viewType.toLowerCase();
+    const capitalizedProperty = property[0].toUpperCase() + property.slice(1);
     const sqlDropView =
-      'DROP VIEW ' + '"' + currentSchema + '"' + '.' + '"' + viewName + '"';
+      'DROP ' +
+      viewType +
+      ' "' +
+      currentSchema +
+      '"' +
+      '.' +
+      '"' +
+      viewName +
+      '"';
```

Comment:
Since this is pretty big, string literal might be a better fit:
```js
const sqlDropView = `DROP ${viewType} "${currentSchema}"."${viewName}".
```

Reply:
Done!